app/protected/route.js

```diff
@@ -23,24 +23,28 @@ export default Route.extend(AuthenticatedRouteMixin, {
   tourManager: service("tour-manager"),
   routing: service("-routing"),
   media: service("media"),
+  store: service(),
+  ajax: service(),
 
   async model() {
-    const id = this.get("session.data.authenticated.user_id");
+    const user = await this.ajax.request("/api/v1/users/me", {
+      method: "GET"
+    });
 
-    const user = await this.store.findRecord("user", id, {
+    const usermodel = await this.store.findRecord("user", user.data.id, {
```

Comment:
Don't you have that ID in the session? I don't think the `/api/v1/me` call is necessary